mage_ai/server/data/base.py

```diff
@@ -46,30 +47,43 @@ def read_parquet_file(self, file_name):
         file_path = os.path.join(self.dir, file_name)
         if not os.path.exists(file_path):
             return pd.DataFrame()
-        return pd.read_parquet(file_path, engine='pyarrow',)
+        return pd.read_parquet(file_path, engine='pyarrow')
 
     def write_parquet_file(self, file_name, df):
         df.to_parquet(os.path.join(self.dir, file_name))
 
     def to_dict(self, detailed):
         pass
-    
+
     @classmethod
     def folder_name(cls):
         return cls.__name__
 
+    @classmethod
+    def is_valid_id(cls, id):
+        opts = os.listdir(cls.path_name())
```

Comment:
not sure whether there will be type difference between id and the dir names. let's convert them to int for comparison to avoid the issue.

Reply:
Converted both id and directory names to integers before comparison